lib/rest2.js

```diff
@@ -886,6 +886,24 @@ class RESTv2 {
     })
   }
 
+  /**
+   * Changes the collateral value of an active derivatives position for a certain pair.
+   *
+   * @param {string} symbol
+   * @param {number} collateral
+   * @param {Method?} cb - legacy callback
+   * @return {Promise} p
+   * @see https://docs.bitfinex.com/v2/reference#rest-auth-deriv-pos-collateral-set
+   */
+
+  derivsPositionCollateralSet (symbol = 'tBTCF0:USDF0', collateral, cb) {
```

Comment:
Could we remove the default assignment to `tBTCF0:USDF`? I think its best to let the request fail and return an error response then them accidentally update the wrong derivative position